pkg/llbgraph/testdata/nodejs-webserver-prod.dot

```diff
@@ -0,0 +1,21 @@
+digraph llbgraph {
+"layer_sha256:06f33e5827a64a3d37fbbe5832c3dc6e525ad97f3f735d3b0525b402834ac61b" [label="sha256:06f33e5827a64a3d37fbbe5832c3dc6e525ad97f3f735d3b0525b402834ac61b",shape="box",style="",fillcolor=""]
```

Comment:
this format seems pretty simple :)

Reply:
Yep, the hardest part was to find a way to reconstruct the graph from JSON files :sweat_smile: